packages/components/transfer/src/composables/use-computed-data.ts

```diff
@@ -1,36 +1,39 @@
 import { computed } from 'vue'
 
-import type { TransferProps } from './transfer'
+import type { TransferDataItem, TransferKey, TransferProps } from '../transfer'
 
 export const useComputedData = (props: TransferProps) => {
   const propsKey = computed(() => props.props.key)
```

Comment:
same with [#21 keyProp](packages/components/transfer/src/composables/use-check.ts)

Perhaps it can be written as a composite function using, eg.

```ts
export const useProps = (props: TransferProps) => {
  const initProps: Required<TransferPropsAlias> = {
    label: 'label',
    key: 'key',
    disabled: 'disabled',
  }

  return computed<Required<TransferPropsAlias>>(() => {
    return Object.assign(initProps, props.props)
  })
}
```

Reply:
At the same time, the `props` are unified within the component. So, we do not need to pass values ​​to `transfer-panel` component by `props`.

and then line 10, 17, 24 can be remove `!`.

Reply:
Thanks for your careful review, I agree with your suggestion! 
IMO, it seems more convenient to change to this:

```ts
export const usePropsConfig = (props: TransferProps) => {
  const initProps: Required<TransferPropsAlias> = reactive({
    label: 'label',
    key: 'key',
    disabled: 'disabled',
    ...props.props,
  })

  return toRefs(initProps)
}
```
How do you feel about that?

Reply:
that is okay, but i suggest `const initProps = reactive<Required<TransferPropsAlias>>`.

The name can be changed as needed.

Reply:
You are right! I missed this.